### PR TITLE
fix: add #okta-sign-in css prop for ie only

### PR DESCRIPTION
**Pull request by sherizada-okta**
## Description:
scope `display: block` to IE browser

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [x] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
IE 11
Screen Shot 2021-01-12 at 6 00 46 PM
Chrome
Screen Shot 2021-01-19 at 3 32 14 PM

### Issue:

- [OKTA-351886](https://oktainc.atlassian.net/browse/OKTA-351886)

### CODE REVIEW

File: assets/sass/okta-theme.scss [hunk 1]
```diff
@@ -81,8 +81,10 @@
 }
 
 #okta-sign-in.auth-container {
-  // for IE to support main tag's width
-  display: block;
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```
Comment (haishengwu-okta) [inline assets/sass/okta-theme.scss:84]: curious why this change?

Comment (sherizada-okta) [reply assets/sass/okta-theme.scss:84]: Remember we had a conversation on slack with Pradeep about applying this css to IE only because it was only needed for IE

**Comment by sherizada-okta** [comment]
@pradeepdewda-okta this is ready for a review

File: assets/sass/okta-theme.scss (hunk 1 above)
Comment (pradeepdewda-okta) [reply assets/sass/okta-theme.scss:84]: So it applies only when it's IE

Comment (pradeepdewda-okta) [reply assets/sass/okta-theme.scss:84]: Good to put a screenshot from chrome or firefox as well..

Comment (sherizada-okta) [reply assets/sass/okta-theme.scss:84]: added a screenshot for Chrome

Comment (sherizada-okta) [reply assets/sass/okta-theme.scss:84]: > So it applies only when it's IE ?

Yes

Comment (pradeepdewda-okta) [reply assets/sass/okta-theme.scss:84]: cool.. It seems this is something we missed in last HF we did..

**Review by pradeepdewda-okta — APPROVED** [review]

File: assets/sass/okta-theme.scss (hunk 1 above)
Comment (sherizada-okta) [reply assets/sass/okta-theme.scss:84]: It was your ask post hotfix. Am I ok to merge this in or do I need to wait for the monthly release? It's not a bugfix or FF gated, so I believe that means monthly release, but this is transparent to users.

Comment (pradeepdewda-okta) [reply assets/sass/okta-theme.scss:84]: yeah you can merge to master.. it's SIW, it's anyway going to be released later.
